cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh

```diff
@@ -37,6 +38,23 @@ function deploy_sriov_operator {
   popd
 }
 
+function deploy_network_resource_injector {
+  webhook_path=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/network-resources-injector-${INJECTOR_GIT_HASH}
+  if [ ! -d $webhook_path ]; then
+    curl -L https://github.com/intel/network-resources-injector/archive/${INJECTOR_GIT_HASH}/network-resources-injector.tar.gz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/
```

Comment:
Can we download this beforehand and put needed files into `cluster-up/cluster/kind-k8s-sriov-1.14.2/manifests`?
On the one hand I don't think it's a good idea to modify files in cluster-up during runtime, because projects importing kubevirtci might check if cluster-up is still the same as in this repository (at least kubevirt does, and redownloads cluster-up if it detects modifications).
And on the other hand we seen it too often that CI fails because of failed downloads, either because of temporary networks issues, or because files not being availabel anymore. Could be easily prevented.
WDYT?

Reply:
Well, the checkout is done in `KUBEVIRTCI_CONFIG_PATH` which should not suffer from the modification issue you mentioned.

However, putting in place some sync mechanism similar to what kubevirt does for CI would not certainly hurt and will make tests run faster. Will work on it.

Reply:
Ah sorry, right, this is `_ci-configs` not `cluster-up`.
However, the issue with downloading stuff during CI runtime remains.